src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -175,6 +175,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="$(MicrosoftVisualStudioSDKEmbedInteropTypesVersion)" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.StaticReviews.Embeddable" Version="$(MicrosoftVisualStudioStaticReviewsEmbeddableVersion)" />
```

Comment:
We do need to confirm this is actually embedding. NuGet has a long history of this acting strangely so let's double check here. If it's not you'll either end up with an RPS regression or potentially a failure to load Roslyn if something goes bad that way.

Reply:
>  If it's not you'll either end up with an RPS regression

I have no issue with that.  If we get an rps regression, we'll take it.  This is a high pri feature we want.

Reply:
And oh this isn't just an RPS concern: this is also a correctness concern; if this doesn't work we might end up crashing without our dependency installed... :smile:

Reply:
Agree with Jason, we will not ship the dll since we are embedding it so if it is not embedded it will be a runtime failure. See https://github.com/github/VisualStudio/pull/2128/files, I had to make an explicit entry to project file to ensure reference was embedded.

Reply:
Done.